server/utils/nunjucksSetup.test.ts

```diff
@@ -0,0 +1,123 @@
+import cheerio from 'cheerio'
+import nunjucks, { Template } from 'nunjucks'
+import { registerNunjucks } from './nunjucksSetup'
+
+describe('Nunjucks Filters', () => {
+  let compiledTemplate: Template
+  let viewContext: Record<string, unknown>
+
+  const njkEnv = registerNunjucks()
+
+  describe('initialiseName', () => {
```

Comment:
Got these tests for this filter in the template app from another project; figured we may as well have it...